### ISSUE TITLE
when use --predictive_mask, got blank mask

### ISSUE THREAD

**Issue by sunnyHelen** [nianticlabs/monodepth2#64]
![image](https://user-images.githubusercontent.com/28101548/63410979-deb28e80-c437-11e9-8359-f2738cc11a1d.png)
When I try to use the --predictive_mask, I got the blank mask.

**Comment by mdfirman** [comment]
Thanks for the bug report. How long did you let the model train for? The predictive mask shouldn't start to produce anything for a while.

**Comment by sunnyHelen** [comment]
I've trained more than 16 epochs. And it seems that it was not blank at the very first. After a while, it started to become totally blank.

**Comment by mdfirman** [comment]
Ok, thanks. As this isn't part of our algorithm we didn't check this in the final code implementation. Perhaps there is some visualisation or initialisation issue going on. I'm out of the office for another two weeks, but when I'm back in I can train a model with it and see if I can reproduce the bug.

**Comment by sunnyHelen** [comment]
Sure, thanks a lot.

**Comment by sunnyHelen** [comment]
Hi, sorry for the late response. I found even with your default parameters on kitti, I still got the blank mask.

**Comment by mdfirman** [comment]
Hi @sunnyHelen , I'm just returning to this now. I have managed to train a model to 1,500 steps with some non-blank masks. Here are some results:

![image](https://user-images.githubusercontent.com/6114258/64126157-a3538080-cda4-11e9-96b1-83389d5a8518.png)

The command I used for this was:

```
python train.py --model_name predictive_mask --predictive_mask --disable_automasking
```

Did you also disable automasking? We have never tried training with both automasking and predictive masks on at once, I expect they would interfere with each other. I will add some code to guard against this untested combination of options.
